lib/tsconfig/4.9.x/ng15/README.md

```diff
@@ -0,0 +1,20 @@
+<h1 align="center">
+   code style - TypeScript 4.9.x + Angular Angular 15.1.x || 15.2.x
+</h1>
+
+## About
+
+This sub-package hosts the [TypeScript](https://www.typescriptlang.org/) configuration for TypeScript 4.9.x. and Angular 15.1.x || 15.2.x
```

Comment:
The version `4.9` can be used all the 15.x versions of Angular. Please adapt to `and Angular 15`.